lib/resoto-helm-chart-stack.ts

```diff
@@ -11,6 +11,38 @@ export class ResotoHelmChartAddOn implements ClusterAddOn {
 
     const cluster = clusterInfo.cluster;
 
+    const stack = clusterInfo.getResourceContext().scope;
+
+    const openIdConnectProviderIssuer = cluster.openIdConnectProvider.openIdConnectProviderIssuer;
+    const saNamespace = 'default';
+    const saAccountName = 'resoto-helm-chart-sa';
+
+    const openIdProviderCondition = new CfnJson(stack, 'openIdProviderCondition', {
+      value: {
+        [`${openIdConnectProviderIssuer}:sub`]: `system:serviceaccount:${saNamespace}:${saAccountName}`
+      }
+    });
+
+    const resotoServiceAccountIamRole = new iam.Role(stack, 'ResotoServiceAccountIamRole', {
```

Comment:
Can you also define the name?
The IAM role in the current stack is named: `resoto-eks-stack-test-ResotoServiceAccountIamRole1-PUJJLU1BJU5T`

Reply:
done